src/components/TodoModal/TodoModal.tsx

```diff
@@ -2,11 +1,31 @@
+import React, { useEffect, useState } from 'react';
 import { Loader } from '../Loader';
+import { User } from '../../types/User';
+
+import { Todo } from '../../types/Todo';
+import { getUser } from '../../api';
+
+type Props = {
+  currentTodo: Todo;
+  onCurrentTodo: (todo: Todo | null) => void;
+};
+
+export const TodoModal: React.FC<Props> = ({ currentTodo, onCurrentTodo }) => {
+  const [user, setUser] = useState<User | null>(null);
+  const [isLoading, setIsLoading] = useState(false);
+
+  useEffect(() => {
+    setIsLoading(true);
+    getUser(currentTodo.userId)
+      .then(setUser)
+      .finally(() => setIsLoading(false));
+  }, [currentTodo.userId]);
```

Comment:
It seems like you're directly interacting with the DOM inside a React component, which is not recommended. Try to avoid this and use React as much as possible.